cloudevents/working-drafts/xml-format.md

```diff
@@ -135,7 +139,7 @@ Example:
 ### 3.3 XML Data
 
 XML data MUST be carried in an element with a defined type of `xs:any` with
-a single child XML element (with any mandatory namespace definitions).
+exactly one child XML element (with any mandatory namespace definitions).
```

Comment:
This probably isn't strictly required, but feels like it gives more of a nudge to SDK authors to validate it.